.nvmrc

```diff
@@ -0,0 +1 @@
+v16.6.0
```

Comment:
why are we going with this node version? any specific reason? why not the LTS?

Reply:
Will move to LTS version

Reply:
fixed